docs/source/references/glossary.md

```diff
@@ -4,50 +4,128 @@
 
 Definitions of all Iroha-related entities can be found here.
 
-## DEX
+# Asset
```

Comment:
```suggestion
## Asset
```
Also making Table of Contents would help.
Currently the preview is as follows:

![Screenshot from 2021-11-23 20-45-14](https://user-images.githubusercontent.com/49983831/143018746-d2cff157-c138-4d48-8bef-491fe2c36c9c.png)

I can see double Triggers